components/doc-wrapper/index.js

```diff
@@ -4,15 +4,18 @@ import { useMemo } from "react"
 
 import { DocWrapperStyled } from "./styles"
 
-const DocWrapper = ({ children, preview }) => {
+const DocWrapper = ({ children, preview, styled }) => {
+  const isStyled = styled ?? true
```

Comment:
Could remove this. and just do the below check with
`if (styled)`

Reply:
@jamespohalloran Should be fixed now